cmd/server.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright © 2022 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/rawdaGastan/pkid/internal"
+	"github.com/rs/zerolog"
+	"github.com/spf13/cobra"
+)
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "pkid",
+	Short: "A command to start the pkid server",
+	Run: func(cmd *cobra.Command, args []string) {
+		logger := zerolog.New(os.Stdout).With().Logger()
+
+		filePath, err := cmd.Flags().GetString("file")
+		if err != nil {
+			logger.Error().Msg("start pkid server failed with error: " + fmt.Sprint(err))
+			return
+		}
+
+		if filePath == "" {
+			logger.Error().Msg("start pkid server failed with error: no file path provided")
+			return
+		}
+
+		port, err := cmd.Flags().GetInt("port")
+		if err != nil {
+			logger.Error().Msg("start pkid server failed with error: " + fmt.Sprint(err))
```

Comment:
![image](https://user-images.githubusercontent.com/64129/202389765-27507982-669a-4fc1-9757-11cc2d4df2be.png)

can use Sprint with all arguments and it will concatenate them, but typically `fmt.Sprintf` will just do fine